test/mangling_upsample.spt

```diff
@@ -0,0 +1,33 @@
+119734787 65536 393230 12 0 
+2 Capability Addresses 
+2 Capability Kernel 
+2 Capability Int64 
+5 ExtInstImport 1 "OpenCL.std"
+3 MemoryModel 2 2 
+10 EntryPoint 6 4 "_ZTSZ4mainE11fake_kernel"
+13 String 11 "kernel_arg_type._ZTSZ4mainE11fake_kernel."
+3 Source 4 100000 
+4 Name 5 "entry"
+6 Name 10 "call2.i.i.i.i"
+4 TypeInt 6 32 0 
+4 TypeInt 9 64 0 
+4 Constant 6 7 0 
+4 Constant 6 8 1 
+2 TypeVoid 2 
+3 TypeFunction 3 2 
+
+
+5 Function 2 4 0 3 
+
+2 Label 5 
+7 ExtInst 9 10 1 s_upsample 7 8 
+1 Return 
+
+1 FunctionEnd 
+
+; RUN: spirv-val %t.spv
```

Comment:
```suggestion
; RUN: spirv-val %s.spv
```

Additionally, this command must be launched after `llvm-spirv %s -to-binary -o %t.spv`

Reply:
Done